queries/reports/groups.js

```diff
@@ -0,0 +1,88 @@
+const GovernmentDomains = {
```

Comment:
Dito as for filters.js, a short comment here to clarify that these functions do not generate complete SPARQL queries, but rather parts of the query generated in queries/reports/index.js would help to make the code easier to understand.